sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/RoslynProfileExporter.java

```diff
@@ -348,7 +348,7 @@ private static void appendLine(Writer writer, String line) {
       writer.write(line);
       writer.write("\r\n");
     } catch (IOException e) {
-      Throwables.propagate(e);
+      throw new RuntimeException(e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00112)